orderby_parser.go

```diff
@@ -30,18 +30,20 @@ func (p *ExpressionParser) ParseOrderByString(orderby string) (*GoDataOrderByQue
 
 	for _, v := range items {
 		v = strings.TrimSpace(v)
-		var order string = ASC
-		if i := strings.LastIndex(v, " "); i > 0 {
-			o := strings.ToLower(v[i+1:])
-			if o == ASC {
-				order = ASC
-			} else if o == DESC {
-				order = DESC
-			} else {
-				return nil, BadRequestError("Invalid sort directive. Must be 'asc' or 'desc'")
-			}
-			v = strings.TrimSpace(v[:i])
+		var order string
+		vLower := strings.ToLower(v)
+		if strings.HasSuffix(vLower, ASC) {
```

Comment:
I think we need to add a space. There could be the weird scenario when the sort field is named `Asc`. For example:
`$orderby=Asc Desc`
`$orderby=Asc Asc`
`$orderby=Asc`

Reply:
Good point. Pushed fix.

Reply:
How about adding these four cases in the unit test?